base/pkg/pkg.jl

```diff
@@ -227,23 +233,32 @@ on all installed or updated packages.
 build(pkgs::AbstractString...) = cd(Entry.build,[pkgs...])
 
 """
-    test()
+    test(; coverage=false)
 
 Run the tests for all installed packages ensuring that each package's test dependencies are
 installed for the duration of the test. A package is tested by running its
 `test/runtests.jl` file and test dependencies are specified in `test/REQUIRE`.
+Coverage statistics for the packages may be generated by passing `coverage=true`.
+The default behavior is not to run coverage.
 """
 test(;coverage::Bool=false) = cd(Entry.test; coverage=coverage)
 
 """
-    test(pkgs...)
+    test(pkgs...; coverage=false)
 
 Run the tests for each package in `pkgs` ensuring that each package's test dependencies are
 installed for the duration of the test. A package is tested by running its
 `test/runtests.jl` file and test dependencies are specified in `test/REQUIRE`.
+Coverage statistics for the packages may be generated by passing `coverage=true`.
+The default behavior is not to run coverage.
 """
 test(pkgs::AbstractString...; coverage::Bool=false) = cd(Entry.test,AbstractString[pkgs...]; coverage=coverage)
 
+"""
+    dependents(packagename)
+
+List the packages that have `packagename` as a dependency.
+"""
 dependents(packagename::AbstractString) = Reqs.dependents(packagename)
```

Comment:
This is probably a good candidate for moving to PkgDev, last time this came up. No harm in documenting it while it's still here though.